### ISSUE TITLE
Switch to JQuery v3

### ISSUE THREAD

**Issue by reckart** [webanno/webanno#1397]
**Describe the refactoring action**
Make Wicket use JQuery v3.

**Expected benefit**
Use non-deprecated dependencies.

**Dependencies**
* blocked by https://github.com/sebfz1/wicket-jquery-ui/issues/311

**Comment by reckart** [comment]
Switching to JQuery 3 causes trouble with Kendo Datasources. It seems that in some cases, widgets such as the ComboBox are initialized with a LocalTransport (i.e. a datasource based on a JSON array present on the page) and (may) only be later switched to a remote transport (which loads the data from the application using XHR). It is not particularly clear to me why this happens with JQuery 3 and not with 2, but it does.

Reproduce:
* open e.g. the a document from the de demo project
* select a POS annotation - force-reload the page and clear caches
* select a named entity annotation 
* observe that the dropdown of the named entity combobox says "no data"

**Comment by reckart** [comment]
Closing this - can re-open or create a new issue once upstream supports JQuery 3.
